Content.Server/Doors/Components/AirlockComponent.cs

```diff
@@ -85,6 +85,19 @@ public bool BoltsDown
 
         private bool _boltLightsWirePulsed = true;
 
+        private bool _emergencyAccess;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public bool EmergencyAccess
+        {
+            get => _emergencyAccess;
+            set
+            {
+                _emergencyAccess = value;
+                UpdateEmergencyLightStatus();
+            }
+        }
```

Comment:
Just so that airlock access/permissions can be predicted eventually, this stuff should all be moved to `SharedAirlockComponent` instead of the server side one.

Also functionality like `ToggleEmergencyAccess()` and calling `UpdateEmergencyLightStatus()` when it is set should just be done in `SharedAirlockSystem` rather than a component. Airlocks should have all their logic moved to systems eventually, and adding this stuff to the component would just make it harder later on.